<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShadersGUI</class>
 <widget class="QMainWindow" name="ShadersGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShadersGUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QDialogButtonBox" name="button_CloseWindow">
      <property name="toolTip">
       <string>Closes the window.</string>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Close</set>
      </property>
      <property name="centerButtons">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="Settings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="1">
         <widget class="QCheckBox" name="value_AutoEnable">
          <property name="toolTip">
           <string>If this is enabled, the effect will be enabled on log in, you can still disable the effect using the keyboard shortcut.

You can use the Whitelist or Blacklist in conjunction with this.</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_Blacklist">
          <property name="toolTip">
           <string>Applications in this list will not be processed.
Seperate applications by comma.</string>
          </property>
          <property name="text">
           <string>Blacklist</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPlainTextEdit" name="value_Whitelist">
          <property name="toolTip">
           <string>Applications outside of this list will not be processed.
Seperate applications by a comma.
Useful if you use &quot;Enabled By Default&quot;.</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_ShaderPath">
          <property name="toolTip">
           <string>The path where the shader files are stored, must be the real full path.</string>
          </property>
          <property name="text">
           <string>Shader Path</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPlainTextEdit" name="value_ShaderPath">
          <property name="toolTip">
           <string>The path where the shader files are stored, must be the real full path.</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QCheckBox" name="value_AutoSave">
          <property name="toolTip">
           <string>On the Shaders and Order tabs, if you change a value and this is enabled, it is applied instantly, without clicking the Save button.</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPlainTextEdit" name="value_Blacklist">
          <property name="toolTip">
           <string>Applications in this list will not be processed.
Seperate applications by comma.</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_Whitelist">
          <property name="toolTip">
           <string>Applications outside of this list will not be processed.
Seperate applications by a comma.
Useful if you use &quot;Enabled By Default&quot;.</string>
          </property>
          <property name="text">
           <string>Whitelist</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_AutoEnable">
          <property name="toolTip">
           <string>If this is enabled, the effect will be enabled on log in, you can still disable the effect using the keyboard shortcut.

You can use the Whitelist or Blacklist in conjunction with this.</string>
          </property>
          <property name="text">
           <string>Auto Enable</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_AutoSave">
          <property name="toolTip">
           <string>On the Shaders and Order tabs, if you change a value and this is enabled, it is applied instantly, without clicking the Save button.</string>
          </property>
          <property name="text">
           <string>Auto Save</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QDialogButtonBox" name="button_SettingsSave">
          <property name="standardButtons">
           <set>QDialogButtonBox::Save</set>
          </property>
          <property name="centerButtons">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Status">
       <attribute name="title">
        <string>Status</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QLabel" name="value_ShadersEnabled">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_ShadersEnabled">
          <property name="text">
           <string>Shaders Enabled:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Shaders">
       <attribute name="title">
        <string>Shaders</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="1">
         <widget class="QDialogButtonBox" name="button_ShadersSave">
          <property name="toolTip">
           <string>Save changes to the file.</string>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Save</set>
          </property>
          <property name="centerButtons">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTableWidget" name="table_Shaders">
          <property name="toolTip">
           <string>You can left click on the shader name or the status (On / Off) to enable / disable a shader.
You can left click a value and change it, when done you can click apply.
To save the changes to the settings file, click save.
If you have made changes but not saved anything, the Save button will display a star.
You can find the description of value by leaving your mouse over it.</string>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>200</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Order">
       <attribute name="title">
        <string>Order</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="0">
         <widget class="QPushButton" name="button_MoveShaderUp">
          <property name="toolTip">
           <string>Move a shader up.</string>
          </property>
          <property name="text">
           <string>Up</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="button_MoveShaderDown">
          <property name="toolTip">
           <string>Move a shader down.</string>
          </property>
          <property name="text">
           <string>Down</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QListWidget" name="value_ShaderOrder"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QDialogButtonBox" name="button_OrderSave">
          <property name="toolTip">
           <string>Save current order to file.</string>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Save</set>
          </property>
          <property name="centerButtons">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
